src/filters/alpha/AlphaFilter.js

```diff
@@ -0,0 +1,45 @@
+import * as core from '../../core';
+import { readFileSync } from 'fs';
+import { join } from 'path';
+
+/**
+ * Simplest filter - applies alpha. Very handy.
```

Comment:
This description is missing two pieces of information:

1). Should have a note about how this differs from Container's `alpha`. e.g., You want to use this instead of Container's alpha property to avoid visual layering of individual elements. AlphaFilter applies alpha evenly across the entire display object and any elements it contains. 

2). VoidFilter had a purpose which was sort of a hack to apply a filter area to something. The "very handy" part... need to explain why.

Reply:
is this better?